src/test/java/com/hocs/test/glue/DocumentsStepDefs.java

```diff
@@ -35,8 +41,11 @@ public void iAmViewingACaseWithDocumentsAttached(String arg0) {
             case "PREVIEWABLE":
                 break;
             default:
-                fail(arg0
-                        + " is not defined in DocumentsStepDefs.iAmViewingACaseWithDocumentsAttached()");
+                System.out.println(arg0
```

Comment:
the string arg0 should have a better name here
public void iAmViewingACaseWithDocumentsAttached(String arg0)